package.json

```diff
@@ -45,7 +45,7 @@
     "@helium/proto-ble": "^4.0.0",
     "@helium/react-native-sdk": "^1.0.0",
     "@helium/transactions": "^4.3.1",
-    "@helium/wallet-link": "^4.6.1",
+    "@helium/wallet-link": "^4.6.2",
```

Comment:
Do we need to bump the other helium-js libraries?

Reply:
I don't think so, but doesn't hurt to.